book/SUMMARY.md

```diff
@@ -16,8 +16,6 @@
   - [Deploy and Submit](/getting-started/deploy-and-submit.md)
     - [iOS - CocoaPods](/getting-started/ios-podspec.md)
     - [Android - Bintray and CircleCI](/getting-started/set-up-bintray-and-circle-ci.md)
-    - [Zapp Plugins Manifest](/zappifest/plugins-manifest-format.md)
```

Comment:
Since you changed the location. please search for broken links and change it as well.